tests/strategies/test_parse.py

```diff
@@ -60,3 +68,55 @@ def test_cuds_parse(repo_dir: "Path") -> None:
     graph_comparison = graph_diff(graph_from_strategy, deser_graph)
     assert graph_comparison[1].serialize() == "\n"
     assert graph_comparison[2].serialize() == "\n"
+
+
+def test_cuds_parse_w_otelib(repo_dir: "Path") -> None:
+    """Test `application/cuds` parse strategy ."""
+    from oteapi.datacache import DataCache
+
+    # Create the otelib client with python as backend
+    from otelib import OTEClient
+    from rdflib import Graph
+    from rdflib.compare import graph_diff
+
+    from dlite_cuds.strategies.parse import CUDSParseStrategy
+
+    client = OTEClient("python")
+
+    # Specify and run the parse strategy as a pipeline in otelib
+    ontologypath = repo_dir / "tests" / "ontologies" / "chemistry.ttl"
+    cudspath = repo_dir / "tests" / "inputfiles_cuds2dlite" / "cuds.ttl"
+
+    config = {
+        "downloadUrl": cudspath.as_uri(),
+        "mediaType": "application/CUDS",
+        "configuration": {
+            "ontologyUrl": ontologypath.as_uri(),
+        },
+    }
+
+    source_and_parse = client.create_dataresource(**config)
+    pipeline = source_and_parse
+    data = pipeline.get()
+
+    # Get the data
+    parsed_data = eval(data.decode("utf-8"))
+
+    cache = DataCache()
+    graph_from_strategy = Graph()
+    graph_from_strategy.parse(
+        data=cache.get(parsed_data["graph_cache_key"]), format="json-ld"
+    )
+
+    # Parse graph directly from the files for comparison
+    # Going through serialisation/deserialisation step required for type specification
+    graph = Graph()
+    graph.parse(ontologypath)
+    graph += graph.parse(cudspath)
+    ser_graph = graph.serialize(format="json-ld")
+    deser_graph = Graph()
+    deser_graph.parse(data=ser_graph, format="json-ld")
+
+    graph_comparison = graph_diff(graph_from_strategy, deser_graph)
+    assert graph_comparison[1].serialize() == "\n"
```

Comment:
This assert statement fails on windows since the return value of `graph_comparison[1].serialize()` is

```
@prefix ns1: <http://emmo.info/emmo#> .

<http://onto-ns.com/ontology/chemistry#ChemicalReaction> ns1:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 """A process that results in the interconversion of chemical species. Chemical reactions may be elementary reactions or stepwise reactions. (It should be noted that this definition includes experimentally observable interconversions of conformers.) Detectable chemical reactions normally involve sets of molecular entities as indicated by this definition, but it is often conceptually convenient to use the term also for changes involving single molecular entities (i.e. \'microscopic chemical events\').

- IUPAC Gold Book"""@en .

```

and not a single newline. 

I have not checked why this is the case on Windows, but it might give a hint on how to fix it.

To avoid issues with blanks (like newlines), I will suggest to stripping them off before comparing.
```suggestion
    assert graph_comparison[1].serialize().strip() == ""
```